### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by jdc308** [PennStateCS/AFCT#70]
We need to “scan” or otherwise analyze the project for accessibility. Then make any easy, low hanging fruit, changes to improve it. The project will never be 100% accessible but any easy changes should be made sooner than later. In a lot of cases it’s probably things like missing descriptive tags.
